adal/build.gradle

```diff
@@ -78,6 +79,45 @@ dependencies {
     testCompile 'junit:junit:4.12'
 }
 
+
+task createPom  {
+    pom {
+        project {
+            groupId 'com.microsoft.aad'
+            artifactId 'adal'
+            version = project.version
+            packaging 'aar'
+            name 'adal'
+
+            description 'Azure active directory library for Android gives you the ability to add Windows Azure Active Directory authentication to your application with just a few lines of additional code. Using our ADAL SDKs you can quickly and easily extend your existing application to all the employees that use Windows Azure AD and Active Directory on-premises using Active Directory Federation Services, including Office365 customers.'
+            url 'https://github.com/MSOpenTech/azure-activedirectory-library-for-android'
+
+            developers {
+                developer {
+                    id 'msopentech'
+                    name 'Microsoft Open Technologies, Inc.'
+                }
+            }
+
+            licenses {
+                license {
+                    name 'Apache License, Version 2.0'
+                }
+            }
+            inceptionYear '2003'
```

Comment:
The first release of adal (ADAL 1.0) is on Jul 11, 2014. So I change the inceptionYear to '2014'. @RandalliLama